ast.cpp

```diff
@@ -53,6 +53,11 @@ object::object(){
 
 CType::~CType() {}
 
+NodeFunctionCall::NodeFunctionCall(const string& functionName) {
+    this->functionName = functionName;
+    c_type = new CIntType();
```

Comment:
add a comment that shows that this is assumed for now, that all functions have return type int